### PR TITLE
Added compatibility with Django 3.0

### PR DESCRIPTION
**Pull request by kstopa**
As Django has dropped six utils now it is necessary to install the standalone package when working with Django 3.0.

### CODE REVIEW

**Comment by LuRsT** [comment]
Great, I was just about to write a PR for this!

**Comment by LuRsT** [comment]
The build seems to be failing for odd reasons, but the fix looks good, @dmkoch, is this PR OK to go? Or should we do anything?

**Comment by domdinicola** [comment]
please update the .travis file
would be nice to move this to django 3.0

**Comment by LuRsT** [comment]
@kstopa could you update the .travis file so this can be merged and the fix released?

**Comment by kstopa** [comment]
> @kstopa could you update the .travis file so this can be merged and the fix released?

Workin on it

**Comment by domdinicola** [comment]
@kstopa can't we drop support to py3.4 and py3.5? and update pyflake version

**Comment by soutys** [comment]
tl;dr: Let's drop all these weird combinations of (old) Python and newer Django framework.

More:
According to https://devguide.python.org/devcycle/#end-of-life-branches now (2020+) we should drop support for py27. Django also moved toward newer python versions:

> Django 3.0 supports Python 3.6, 3.7, and 3.8. We highly recommend and only officially support the latest release of each series.
>
> The Django 2.2.x series is the last to support Python 3.5.

(https://docs.djangoproject.com/en/3.0/releases/3.0/)

**Comment by kstopa** [comment]
Hi, I am almost done. On local all the tests passes but on Travis I have:

```
Creating test database for alias 'default'...
Traceback (most recent call last):
File "/home/travis/build/kstopa/django-jsonfield/.tox/py36-master/lib/python3.6/site-packages/django/db/backends/utils.py", line 82, in _execute
   return self.cursor.execute(sql)
File "/home/travis/build/kstopa/django-jsonfield/.tox/py36-master/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 399, in execute
    return Database.Cursor.execute(self, query)
sqlite3.OperationalError: no such table: django_content_type
```

Any suggestions???

**Comment by domdinicola** [comment]
@kstopa a couple of things:
1) https://stackoverflow.com/questions/56875632/upgrade-django-synchro-for-django-2-2-operationalerror-no-such-table-django
2) try add app_label to the meta in the model

**Comment by domdinicola** [comment]
@kstopa what happened?

**Comment by kstopa** [comment]
I'm unable to make it work on travis and I lost to much time trying.

**Comment by kstopa** [comment]
It should work if I can delete from testing content_type from one of the models.
